client/src/components/FestivalMatch.js

```diff
@@ -30,20 +31,25 @@ const FestivalInfo = styled.p`
   margin: 3px 0;
 `;
 
-const selectedGenres = new Array(
-  sessionStorage.getItem('SelectedGenres').split(',')
-);
+let selectedGenres = [];
+if (sessionStorage.getItem('SelectedGenres') == null) {
+  console.log('Oh no!');
+} else {
+  const genres = sessionStorage.getItem('SelectedGenres');
+  selectedGenres = genres.toString().split(',');
+}
+
+const fetchRoute =
+  '/api/festivals?genres_like=' + selectedGenres.join('&genres_like=');
 async function fetchFestivals() {
-  const response = await fetch(
-    '/api/festivals?genres_like' + selectedGenres.join('&genres_like=')
-  );
+  const response = await fetch(fetchRoute);
   const festivals = await response.json();
   return festivals;
 }
 
 function FestivalMatch() {
```

Comment:
Maybe you can add a refresh state or function here so you don't have to refresh on your own (I had the same problem with updating my concert)